net/bully/Makefile

```diff
@@ -0,0 +1,36 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=bully
+PKG_VERSION:=1.1
+PKG_RELEASE:=31
```

Comment:
PKG_RELEASE start with 1, not 31

Reply:
I think the confusion here is that he wants version 1.1 + 31 commits.

Reply:
I took it from here https://github.com/aanarchyy/bully/blob/master/version.mk

Not sure either why 31, but I'll change to 1.